tests/ibc-hooks/ibc_middleware_test.go

```diff
@@ -541,12 +553,13 @@ func (suite *HooksTestSuite) TestFundTracking() {
 // custom MsgTransfer constructor that supports Memo
 func NewMsgTransfer(token sdk.Coin, sender, receiver, channel, memo string) *transfertypes.MsgTransfer {
 	return &transfertypes.MsgTransfer{
-		SourcePort:       "transfer",
-		SourceChannel:    channel,
-		Token:            token,
-		Sender:           sender,
-		Receiver:         receiver,
-		TimeoutHeight:    clienttypes.NewHeight(1, 500),
+		SourcePort:    "transfer",
+		SourceChannel: channel,
+		Token:         token,
+		Sender:        sender,
+		Receiver:      receiver,
+		// TODO: check why this timeout number had to be adjusted
+		TimeoutHeight:    clienttypes.NewHeight(3, 500),
```

Comment:
ditto

Reply:
we adjusted the chain IDs for the tests, which now increments the chain suffix while still keeping the same EIP-155 compliant ID, so instead of going `osmosis1-1`, `osmosis2-1`, ... it's now incrementing `osmosis_9009-1`, `osmosis_9009-2`, etc., which also increases the revision height for the IBC channels